sync.cmd

```diff
@@ -0,0 +1,3 @@
+@if "%_echo%" neq "on" echo off
```

Comment:
hmm not really. It would end up printing the call to the executor. I guess we could delete it